sdks/go/pkg/beam/pardo.go

```diff
@@ -414,7 +412,36 @@ func ParDo6(s Scope, dofn interface{}, col PCollection, opts ...Option) (PCollec
 func ParDo7(s Scope, dofn interface{}, col PCollection, opts ...Option) (PCollection, PCollection, PCollection, PCollection, PCollection, PCollection, PCollection) {
 	ret := MustN(TryParDo(s, dofn, col, opts...))
 	if len(ret) != 7 {
-		panic(fmt.Sprintf("expected 7 output. Found: %v", ret))
+		panic(formatParDoError(dofn, len(ret), 7))
 	}
 	return ret[0], ret[1], ret[2], ret[3], ret[4], ret[5], ret[6]
 }
+
+//formatParDoError is a helper function to provide a more concise error
```

Comment:
Nit: add a space between // and formatParDoError

Reply:
7c612c9db2